app/views/pregnancies/_new_pregnancy.html.erb

```diff
@@ -2,27 +2,35 @@
 <%= bootstrap_form_for pregnancy do |f| %>
 
   <%= f.fields_for patient do |p| %>
-    <div class="col-md-2">
-      <%= p.text_field :primary_phone, label: "Phone Number" %>
+
+
+<div class="col-md-3">
+    <div class="col-md-6">
+      <%= p.text_field :primary_phone, label: "Number" %>
     </div>
-    <div class="col-md-2">
-      <%= p.text_field :name, label: "First and Last Name" %>
+    <div class="col-md-6">
+      <%= p.text_field :name, label: "Name" %>
     </div>
+  </div>
   <% end %>
 
-  <div class="col-md-2">
-    <%= f.select :last_menstrual_period_weeks, options_for_select(weeks_options, pregnancy.last_menstrual_period_weeks ), label: 'LMP at intake', autocomplete: 'off' %>
-  </div>
+  <div class="col-md-3">
 
-  <div class="col-md-2 hide-label">
-    <%= f.select :last_menstrual_period_days, options_for_select(days_options, pregnancy.last_menstrual_period_days ), label: '.', autocomplete: 'off' %>
-  </div>
+    <div class="col-md-6">
+      <%= f.select :last_menstrual_period_weeks, options_for_select(weeks_options, pregnancy.last_menstrual_period_weeks ), label: 'LMP', autocomplete: 'off' %>
+    </div>
+
+    <div class="col-md-6 hide-label">
+      <%= f.select :last_menstrual_period_days, options_for_select(days_options, pregnancy.last_menstrual_period_days ), label: 'LMP', autocomplete: 'off' %>
+    </div>
+
+</div>
 
   <div class="col-md-2">
     <%= f.date_field :initial_call_date, label: "Initial Call Date" %>
   </div>
 
-  <div class="col-md-2">
+  <div class="col-md-2 voicemailOk">
```

Comment:
Let's pop off this voicemail class for now -- I'm inlined to wait until we get some clarity from team UX. Thoughts?